blog/ga-for-mars.adoc

```diff
@@ -0,0 +1,92 @@
+= JBoss Tools and Red Hat Developer Studio for Eclipse Mars
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2015-10-06
+
+http://tools.jboss.org/downloads/jbosstools/mars/4.3.0.Final.html[JBoss Tools 4.3] and http://tools.jboss.org/downloads/devstudio/mars/9.0.0.GA.html[Red Hat JBoss Developer Studio 9] for Eclipse Mars are now generally available!
+
+image::/blog/images/jbds9.png[]
+
+CAUTION: Java 8 is required for installing and using of JBoss Tools. We still support developing and running applications using older Java runtimes. See more in link:/blog/2015-06-23-beta1-for-mars.html#java-8-to-run-eclipse-older-runtimes-ok-for-builds-deployment[Beta1 blog].
```

Comment:
"installation and use" or "installing and using"... don't need the "of" in there.